tools/openqa-run

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -euo pipefail
+
+usage() {
+    cat << EOF
+Usage: openqa-run [OPTIONS...] OPENQA_SERVICE
+Call an openQA service OPENQA_SERVICE with appropriate user account and the
+base-dir set to the local working copy.
+
+Example:
+    openqa-run openqa-webui-daemon
```

Comment:
This kind of usage is very annoying without auto completion. In my helper is worked around this via shortcuts.

The script doesn't seem to ensure that e.g. `openqa-webui-daemon` is actually in the `PATH` so I guess this example also won't lead to a great out-of-the-box experience even if copied verbatimly. (The example you added to the docs does make sense, though.)